docs/state-persistence.md

```diff
@@ -24,7 +24,7 @@ This provided key will be used as the react native `AsyncStorage` key to save th
 
 ### Development Mode
 
-This feature is particularly save and desirable in development mode. You can enable it selectively using the following approach:
+This feature is particularly safe and desirable in development mode. You can enable it selectively using the following approach:
```

Comment:
While it's probably better-sounding, the sentence doesn't make a lot of sense semantically.

I'd just change it to "This feature is particularly useful in development mode." Does that sound better to you?